CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 # ---------------------------------------------------------------------------
 # Author:      jongough aka. Jon Gough Copyright: License:     wxWidgets License
 # ---------------------------------------------------------------------------
-
+# Frontend2 Author: v1.0.139.2  Author: Jon Gough
```

Comment:
Please remove this, it's a maintenance  nightmare. The version is defined in the VERSION_\*
constants.

Reply:
Well, this one is important to me. I need to know what version of Frontend2 is being used in various plugins, so that I can manage them properly.  Therefore I keep track of the testplugin too in this way. 

This approach works for me and I am not going to change it. I know which version of Frontend2 a plugin is using and I also can relate that to the right commit in testplugin.
It saves lots of time and guessing what version of frontend2 is being used.

I don't think I will be changing this very soon.

Reply:
Fair enough, this is about your fork. But it doesn't mean it should be part of the PR, which should be kept simple. As long as you do, you need to instruct whoever merges it how to cope with your local setup. I would not accept it, for sure (Jon's decision is his own).

Still think that a proper PR fixing #109 should be exactly one line, similar to #113. The workflow I described in https://github.com/jongough/testplugin_pi/pull/111#issuecomment-728840343 will achieve this.

This is yet another reason why you need to keep these changes in a separate branch, so you can keep your own stuff but making something simpler in the PR.